templates/js_main.tmpl

```diff
@@ -0,0 +1,26 @@
+/**
+    This binding file was auto generated based on FLIX template v{{.Version}}. 
+    Changes to this file might get overwritten 
+**/
+
+import * as fcl from "@onflow/fcl"
+{{- if .IsLocalTemplate }}
+import flixTemplate from "{{.Location}}"
+{{- else}}
+const flixTemplate = "{{.Location}}"
+{{- end}}
+
+const parameterNames = [
+  {{- range $index, $ele := .Parameters -}}
+     {{if $index}}, {{end}}"{{$ele.Name}}"
+  {{- end -}}
+];
+
+{{if .IsScript}}
+{{- template "js_script.tmpl" .}}
```

Comment:
`js_` doesn't feel specific enough. Should probably be `fcl_`. Because other JS libraries could want to come along and add their templates for their libraries.

Reply:
changed to js_fcl_